src/merkle.rs

```diff
@@ -2007,4 +2032,37 @@ fn test_get_merkle_tree_methods() {
 
     assert!(get_merkle_tree_leafs(1398102, 4).is_err());
     assert!(get_merkle_tree_leafs(299594, 8).is_err());
+
+    let mib = 1024 * 1024;
+    let gib = 1024 * mib;
+
+    // 32 GiB octree cache size sanity checking
+    let leafs = 32 * gib / 32;
+    let rows_to_discard = StoreConfig::default_rows_to_discard(leafs, 8);
+    let tree_size = get_merkle_tree_len(leafs, 8).expect("");
+    let cache_size = get_merkle_tree_cache_size(leafs, 8, rows_to_discard).expect("");
+    assert_eq!(leafs, 1073741824);
```

Comment:
No need to change for this PR necessarily, but would you consider formatting numbers like `1_073_741_824` to make it easier to parse the order of magnitude. In fact, please also consider writing them as powers of 2 (when possible), which makes it even easier to verify correctness and reason about them. For example, if this had been written as `1 << 30` (2^30), I would know immediately that it is the number of 32 (2^5) byte leaves of a 32 GiB (2^32 byte) base row (since 2^30 * 2^5 = 2^35 because 30 + 5 = 35).

If a reader can easily verify that the numbers used *must* be correct, it's much easier to review, to understand the logic later (for those approaching the code), and ensure it is correct in the first place.

This was an outsized comment on a piece of code which is fine as-is. I used it to make a general editorial comment I hope will be useful.